package.cordovabuild.json

```diff
@@ -146,25 +137,19 @@
     "enketo-transformer": "^4.0.0",
     "fast-xml-parser": "^4.2.2",
     "fs-extra": "^9.0.1",
-    "humanize-duration": "3.31.0",
-    "i18next": "^22.5.0",
+    "humanize-duration": "^3.31.0",
+    "i18next": "^23.7.6",
```

Comment:
Although we don't specifically test the `i18nextInit.ts` file, some of the tests do invoke it because those tests depend on i18n

So there is coverage for it downstream